pkg/network/cluster_config.go

```diff
@@ -41,7 +41,7 @@ func ValidateClusterConfig(clusterConfig configv1.NetworkSpec) error {
 		}
 		size, _ := cidr.Mask.Size()
 		// The comparison is inverted; smaller number is larger block
-		if cnet.HostPrefix < uint32(size) {
+		if cnet.HostPrefix <= uint32(size) {
```

Comment:
This was discussed and we decided it was correct as it is; you're allowed to have a cluster network that's only big enough for a single node.